sass/_theme_variables.sass

```diff
@@ -52,6 +52,11 @@ $nav-link-color-visited:              $purple
 $nav-link-color-hover:                lighten($nav-link-color, 6%) !default
 $nav-link-color-alt:                  hsl(33, 100%, 51%)
 
+// Sidebar
+$sidebar-transform-hide:              translateX(-100%)
+$sidebar-transform-show:              translateX(0%)      
+$sidebar-transition:                  transform 0.3s ease
```

Comment:
Transforms and transitions may need prefixed forms as well for IE 9 and older Android and iOS browsers